pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1087,11 +1087,18 @@ func (og *operationGenerator) GenerateUnmapDeviceFunc(
 		glog.V(5).Infof("UnmapDevice: deviceToDetach.DevicePath: %v", deviceToDetach.DevicePath)
 		loopPath, err := og.blkUtil.GetLoopDevice(deviceToDetach.DevicePath)
 		if err != nil {
-			glog.Warningf(deviceToDetach.GenerateMsgDetailed("UnmapDevice: Couldn't find loopback device which takes file descriptor lock", fmt.Sprintf("device path: %q", deviceToDetach.DevicePath)))
+			if err.Error() == util.ErrDeviceNotFound {
+				glog.Warningf(deviceToDetach.GenerateMsgDetailed("UnmapDevice: Couldn't find loopback device which takes file descriptor lock", fmt.Sprintf("device path: %q", deviceToDetach.DevicePath)))
+			} else {
+				errInfo := "UnmapDevice: Couldn't find loopback device which takes file descriptor lock," + fmt.Sprintf("device path: %q", deviceToDetach.DevicePath)
```

Comment:
I'd prefer error message like this;
`"UnmapDevice.GetLoopDevice failed to get loopback device, " + fmt.Sprintf("device path: %q", deviceToDetach.DevicePath)`

Reply:
@mtanino  changed.